### ISSUE TITLE
Add new view in Personnel tab to see abilities listed

### ISSUE THREAD

**Issue by mjbroekman** [MegaMek/mekhq#2251]
In 0.47.14, there are a number of views in the Personnel tab to see different attributes of people in the unit. There is, however, no way to see the abilities that a pilot has without clicking on the pilot to see their information card. It would be exceptionally helpful to have a view that showed the abilities in a list so you could see the abilities across all members of the unit.

**Comment by Windchild292** [comment]
iirc hover over should list the abilities

**Comment by mjbroekman** [comment]
I don't get any popups in the Personnel tab (or any other tabs in MekHQ) on MacOS w/ 0.47.14. Not sure where there _should_ be popups outside of combat.

**Comment by Windchild292** [comment]
There isn't a tooltip for the number of abilities? It might take a second to show. If not, that's a Java or Mac bug and we'll need to dig in from there.

**Comment by mjbroekman** [comment]
Correct. I'm not getting tooltips or other popups in MekHQ on my Mac (MacOS 10.15.7). I get the tooltip info in MegaMek fine, just not in the MekHQ campaign interface.

That being said, having that be a tooltip wouldn't help from the user experience perspective as I wouldn't be able to see all the abilities across all my personnel in a single list.
